daemon/call.c

```diff
@@ -503,6 +522,19 @@ static void call_timer(void *ptr) {
 	atomic64_set(&rtpe_stats.packets, atomic64_get_na(&tmpstats.packets));
 	atomic64_set(&rtpe_stats.errors, atomic64_get_na(&tmpstats.errors));
 
+	/* update statistics regarding requests per second */
+	offers = atomic64_get(&rtpe_statsps.offers);
+	atomic64_set(&rtpe_statsps.offers, 0);
```

Comment:
I suggest using `atomic64_get_set` instead of one `get` and one `set` here, as otherwise there's a chance of losing some counts. (Applies to all 3 occurrences)